js/game.js

```diff
@@ -5,8 +5,75 @@ export default class Game {
     this.tamagotchi = new Tamagotchi();
   }
 
-  start = ({ healthElement, hungerElement, energyElement, funElement }) => {
-    this.tamagotchi.mount({ healthElement });
+  start = ({
+    healthElement,
+    hungerElement,
+    energyElement,
+    funElement
+  }) => {
+    this.tamagotchi.mount({
+      healthElement,
+      hungerElement,
+      energyElement,
+      funElement,
+    });
+
     console.log("Game started");
+
+    const decreaseHealth = () => {
+      if (this.tamagotchi.hunger.value <= 0) this.tamagotchi.health.value--;
+      this.tamagotchi.mount({
+        healthElement,
+        hungerElement,
+        energyElement,
+        funElement,
+      })
+      
+      if (this.tamagotchi.health.value <= 0) {
+        clearInterval(healthId);
+        clearInterval(hungerId);
+        clearInterval(energyId);
+        clearInterval(funId);
+        console.log("You lost")
+      }
+    };
+
+    const decreaseHunger = () => {
+      this.tamagotchi.hunger.value--;
+      this.tamagotchi.mount({
+        healthElement,
+        hungerElement,
+        energyElement,
+        funElement,
+      })
+    };
+
+    const decreaseEnergy = () => {
+      this.tamagotchi.energy.value--;
+      if (this.tamagotchi.fun.value <= 0) this.tamagotchi.energy.value--;
+      this.tamagotchi.mount({
```

Comment:
**nitpick:** We shouldn't have to use this initial mount method when you have specific one available for displaying `energy` parameter. I mean in this case: `this.tamagotchi.displayEnergy(energyElement);`